network-store-server/src/main/java/com/powsybl/network/store/server/Mappings.java

```diff
@@ -671,6 +688,14 @@ private void createTwoWindingsTransformerMappings() {
                 }
                 attributes.getRatioTapChangerAttributes().setRegulationValue(value);
             }));
+        twoWindingsTransformerMappings.addColumnMapping("ratioTapChangerSolvedTapPosition", new ColumnMapping<>(Integer.class,
```

Comment:
To be done for ThreeWindingsTransformers ?

Reply:
done